planetarium/NineChronicles
# Disable RTL Text on FriendInfoPopup's titletext.

### Description

SSIA

### How to test

1. Enter arena and see other player's title text and check if it is properly displayed.

### Required Reviewers

@Namyujeong or @planetarium/9c-dev
